test/test_user_interface.py

```diff
@@ -0,0 +1,17 @@
+# TODO the following need tests
```

Comment:
I agree we need unit tests for this, but I'd rather not put TODO comments into the code. Instead, I prefer to use GitHub issues to track the work that needs to be done. Would you mind removing these and instead creating a new issue for adding the tests? Thanks!

Reply:
Removed unit test files in https://github.com/iansan5653/open-mcr/pull/74/commits/3ee26e7adaf38a1326bca42319a01e0a2eaa4e46